docs/machine-learning/how-to-guides/load-data-ml-net.md

```diff
@@ -161,7 +161,70 @@ Use the `Load` method to load the data into an [`IDataView`](xref:Microsoft.ML.I
 ```csharp
 IDataView data = loader.Load(dbSource);
 ```
+## Load data from images files
 
+Create a model that hold image path and a lebel . `ImagePath` is the absolute path of the image in data source directory. `Label` is the class or category of the actual image file..
+
+```csharp
+ public class ImageData
+        {
+            [LoadColumn(0)]
+            public string ImagePath;
+
+            [LoadColumn(1)]
+            public string Label;
+        }
+```
+## Load the Image Data from input directory.
+
+```csharp
+
+public static IEnumerable<ImageData> LoadImagesFromDirectory(string folder,
+            bool useFolderNameAsLabel = true)
+        {
+            var files = Directory.GetFiles(folder, "*",
+                searchOption: SearchOption.AllDirectories);
+            foreach (var file in files)
+            {
+                if (Path.GetExtension(file) != ".jpg")
+                    continue;
+
+                var label = Path.GetFileName(file);
+                if (useFolderNameAsLabel)
+                    label = Directory.GetParent(file).Name;
+                else
+                {
+                    for (int index = 0; index < label.Length; index++)
+                    {
+                        if (!char.IsLetter(label[index]))
+                        {
+                            label = label.Substring(0, index);
+                            break;
+                        }
+                    }
+                }
+
+                yield return new ImageData()
+                {
+                    ImagePath = file,
+                    Label = label
+                };
+
+            }
+        }
+ ```
+
+ Load image
+ ```csharp
+ //Create MLContext
+MLContext mlContext = new MLContext();
+
+ IEnumerable<ImageData> images = LoadImagesFromDirectory(
+                folder: pathe, useFolderNameAsLabel: true);
```

Comment:
Made it more generic since `path` wasn't defined in the code snippet. 

```suggestion
 IEnumerable<ImageData> images = LoadImagesFromDirectory(
                folder: "your-image-directory-path", useFolderNameAsLabel: true);
```